expertise/service/celery_tasks.py

```diff
@@ -25,6 +25,7 @@ def update_status(job_dir, new_status, desc=None):
         config['description'] = descriptions[new_status]
     else:
         config['description'] = desc
+    config['mdate'] = int(time.time())
```

Comment:
We save the timestamps in milliseconds. I think this is saving it in seconds though

Reply:
Ah I see an example of this is the openreview-py repo, this is a simple fix